docs/develop/index.md

```diff
@@ -2,31 +2,15 @@
 
 **Intended audience**
 
-This guide is intended both for developers that needs to develop new features for their own InvenioRDM instance as well as core InvenioRDM developers.
+This guide is intended for developers that need to develop new features for their own InvenioRDM instance.
 
 ## Getting started
 
 Explains how to get development install up and running:
 
-- [Source code](getting-started/source-code.md)
-- [Package development](getting-started/package-development.md)
-- [Instance development](getting-started/instance-development.md)
-- [Debugging](getting-started/debugging.md)
-- [Coding style](getting-started/code-style.md)
 - [Virtual environments](getting-started/virtualenvs.md)
+- [Debugging](getting-started/debugging.md)
```

Comment:
Is it OK to remove all these links?

Reply:
I moved those links here: https://github.com/inveniosoftware/docs-invenio-rdm/pull/746/files/d50b47113ebb74f97439de50216ca709a3d82b98#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R243 . After looking at their content, I assessed that they were mostly for people contributing to the InvenioRDM codebase. The remaining ones are more general / apply to people customizing/working on their own instance, so I left there.

The plan is to transform the "Develop" section into ones 
- more for operators of an instance
- more for maintainers
...   
in subsequent reorganizations (from original prototypes + discussion during the DWG)
 
If I missed a file, let me know!

Reply:
The Getting started section in Develop now looks like: https://fenekku.github.io/docs-invenio-rdm/develop/ . There is virtualenvs, Debugging and Getting Help sub-sections in there. We'll move those and flesh them out when dealing with the new sections mentioned above. I think they are still appropriate for "developing on your instance" for now.